components/datepicker/datepicker.stories.js

```diff
@@ -0,0 +1,89 @@
+import { createTemplateFromVueFile } from '@/common/storybook_utils';
+import { action } from '@storybook/addon-actions';
+import DtDatepicker from './datepicker.vue';
+import DtDatepickerTemplate from './datepicker_default.story.vue';
```

Comment:
```suggestion
import DtDatepickerDefaultTemplate from './datepicker_default.story.vue';
```

Just in case we need to import another templates for variants or something like so.

Reply:
Not necessary to do it now but I will added it